packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -1870,6 +1870,14 @@ describe('<Autocomplete />', () => {
       expect(handleChange.callCount).to.equal(1);
       expect(handleChange.args[0][1]).to.equal('あ');
     });
+
+    it('should only render endAdornment when hasClearIcon or hasPopupIcon is true', () => {
```

Comment:
Better to not include implementation details in title. 
```suggestion
    it('should render endAdornment only when clear icon or popup icon is available', () => {
```